eigen_cpp/lib/eigen_spmat_c.cpp

```diff
@@ -3,9 +3,13 @@
  * Copyright (c) 2016-2017 Liang Wang <liang.wang@cl.cam.ac.uk>
  */
 
+#include "sparse_solver.h"
+
 /******************** pointer conversion  ********************/
 
 typedef SparseMatrix<spmat_c_elt, Eigen::RowMajor, INDEX> spmat_c;
+typedef SparseMatrix<spmat_c_elt, Eigen::ColMajor,   int> col_major_spmat_c;
```

Comment:
Is there a reason for not using INDEX?

Reply:
Yeah, that was a weird one. Using INDEX broke compilation for me. Maybe it was some issue local to my environment? Can you try changing it to see if you can reproduce that issue?